dspace-spring-rest/src/test/java/org/dspace/app/rest/BitstreamRestRepositoryIT.java

```diff
@@ -587,4 +589,24 @@ public void deleteLogo() throws Exception {
         getClient(token).perform(delete("/api/core/bitstreams/" + col.getLogo().getID()))
                    .andExpect(status().is(422));
     }
+
+    @Test
+    public void patchBitstreamMetadataAuthorized() throws Exception {
+        runPatchMetadataTests(admin, 200);
+    }
+
+    @Test
+    public void patchBitstreamMetadataUnauthorized() throws Exception {
+        runPatchMetadataTests(eperson, 403);
+    }
+
+    private void runPatchMetadataTests(EPerson asUser, int expectedStatus) throws Exception {
+        context.turnOffAuthorisationSystem();
+        parentCommunity = CommunityBuilder.createCommunity(context).withName("Community").withLogo("logo_community")
+                .build();
```

Comment:
We should have a `context.restoreAuthSystemState();` here.  I think we need to get in the habit of only turning off authorization to create test content.  In test it likely doesn't matter that much, but it's more of a safety net that tests which are specifically testing *authorization* aren't accidentally disabling it during important parts of the test.